controllers/openstackdataplanerole_controller.go

```diff
@@ -281,6 +281,14 @@ func (r *OpenStackDataPlaneRoleReconciler) Reconcile(ctx context.Context, req ct
 		instance.Status.Deployed = true
 		r.Log.Info("Set ReadyCondition true")
 		instance.Status.Conditions.Set(condition.TrueCondition(condition.ReadyCondition, dataplanev1beta1.DataPlaneRoleReadyMessage))
+		for _, node := range nodes.Items {
+			node.Status.Conditions.MarkTrue(condition.ReadyCondition, fmt.Sprintf("Deployed by role.Name %s role.Namespace %s", instance.Name, instance.Namespace))
+			node.Status.Deployed = true
+			err = helper.PatchInstance(ctx, &node)
```

Comment:
will this trigger a Reconcile on the OpenstackDataPlaneNode object? If so then that Reconcile might reset the status condition.

Reply:
This [1] makes it seem like updates (patches too?) to status do trigger a reconcile, but it's from 2019.

[1] https://github.com/kubernetes-sigs/kubebuilder/issues/618